addons/l10n_eu_service/wizard/wizard.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import fields, models, api
+from openerp.tools.translate import _
+
+class l10n_eu_service(models.TransientModel):
+
+    """Create fiscal positions for EU Service VAT"""
+
+    _name = "l10n_eu_service.wizard"
+    _description = __doc__
+
+    def compute_default_value_chart(self):
+        user = self.env.user
+        return self.env['account.account'].search(
+            [('company_id', '=', user.company_id.id), ('parent_id', '=', False)], limit=1)
+
+    def compute_default_value_fiscal_position(self):
+        user = self.env.user
+        return self.env['account.fiscal.position'].search(
+            [('company_id', '=', user.company_id.id), ('vat_required', '=', False),
+             ('country_group_id.name', '=', 'Europe')], limit=1)
+
+    def compute_default_value_tax(self):
+        user = self.env.user
+        return self.env['account.tax'].search(
+            [('company_id', '=', user.company_id.id), ('type_tax_use', '=', 'sale'),
+             ('type', '=', 'percent'), ('account_collected_id', '!=', False),
+             ('tax_code_id', '!=', False)], limit=1, order='amount')
+
+    def compute_default_value_done_country(self):
+        user = self.env.user
+        eu_country_group = self.env['res.country.group'].search([('name', '=', 'Europe')])
```

Comment:
Here as well, we could use `ir.model.data.xmlid_to_object('base.europe')`. Also you need to check that `eu_country_group` was indeed found, as the group may have been deleted.